.claude/metadata-updater.sh

```
@@ -94,8 +142,8 @@ update_metadata_key() {
   # Create temp file
   local temp_file="${metadata_file}.tmp"
 
-  # Escape special sed characters in value (& | / \)
-  local escaped_value=$(echo "$value" | sed 's/[&|\/]/\\&/g')
+  # Escape special sed characters in value (& and \ — not | or / in BRE)
+  local escaped_value=$(echo "$value" | sed 's/[&\\]/\\&/g')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)